README.md

```diff
@@ -3,6 +3,7 @@
 # **Decluttarr**
 
 ## Table of contents
+
```

Comment:
This PR is difficult to review, since you also ran a format cleanup, which now marks many files & lines as changed, taking away attention from the relevant changes.

Could you please revert that, so we only see the material changes?